torch/csrc/lazy/core/ir.cpp

```diff
@@ -28,7 +28,8 @@ OpKind OpKind::Get(const std::string& name) {
 }
 
 hash_t OpKind::hash() const {
-  return StringHash(op.toQualString());
+  // op contains only a `c10::unique_t value` field, and unique_t = uint32_t
+  return Hash(static_cast<uint32_t>(op));
```

Comment:
Can we replace `uint32_t ` with `unique_t` in case https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/interned_strings.h#L541 changes the `using ...`?

Reply:
oh yea that's a really good point and i made the change. thanks!